src/pull_request.closed.ts

```diff
@@ -0,0 +1,9 @@
+import { Context } from "probot";
+import { addContributions } from "./add_contribution";
+
+export async function pull_requestClosed(context: Context) {
+  // check if PR is merged
+  if (context.payload.pull_request.merged) {
+    await addContributions(context);
+  }
```

Comment:
We want to add them even if the the PR is not merged

Reply:
Sure will change that to when PR is opened. Will that work?

Reply:
I was initially thinking to do it on PR open, but I think it is better to do it on close because all the files may not be changed on PR open. So it'll be more accurate it to do it on PR close.

Reply:
Sure then will just remove the if condition.

Reply:
Done